src/Action.hpp

```diff
@@ -7,6 +7,7 @@
 
 namespace SokuLib
 {
+    //! @brief Represents all actions a character can do.
```

Comment:
It's technically not all of them. It's also very arbitrary because these can vary a lot depending on the character (yes Suwako I'm looking at you).
[That](https://github.com/SokuDev/SokuMods/blob/3cdbaa6c6558791842564c7347a8f1d39aa8a1a2/modules/AdvancedPracticeMode/Inputs.cpp#L32)'s the list I found for APM but it's probably not fully complete. I think it should have a disclaimer here. Also, some character straight up are missing some moves from here (i.e. 1A or j.1C).